src/main/java/at/hannibal2/skyhanni/utils/CollectionUtils.kt

```diff
@@ -133,6 +133,9 @@ object CollectionUtils {
         return newList
     }
 
+    /**
+     * This does not work inside a [buildList] block
```

Comment:
yes, because build list blocks are MutableLists, this is an extension function to a List.

Reply:
Also it looks like this function is unused?